backend/courses/views.py

```diff
@@ -524,3 +525,46 @@ def delete(self, request):
             res["message"] = "Friendship request already rejected."
             return JsonResponse(res, status=status.HTTP_409_CONFLICT)
         return JsonResponse(res, status=status.HTTP_200_OK)
+
+
+class AuditView(generics.ListAPIView):
+    """
+    This route allows you to list courses that are currently happening with filters to
+    exclude departments, buildings etc. The route will return a list of courses in
+    decreasing course_quality. Without GET parameters, this route will return all
+    currently running courses.
+    """
+
+    def get(self, request, semester):
+        timestamp = request.GET.get("timestamp")
```

Comment:
Nit: DRF (Django rest framework) provides the request.query_params.get(...) which is more readable